### DISCUSSION TITLE
How to deploy Objeck app?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#99] [General]
Does I need to install the Objeck runtime environment (equivalent to the JRE) on the target machine first?

**Comment by nsgnkhibdk2cls0f** [comment]
https://www.objeck.org/getting_started.html#create_portable

**Reply by ghost** [reply]
`>>> Invalid Objeck install directory: '' <<<`

**Reply by nsgnkhibdk2cls0f** [reply]
I don't know. This is what I found on the docs. I have never think about deploying something. I'm still setting up text editors for syntax highlighting before learning the language. I'm in a too early stage to think about it. I'm not a programmer too, I don't think I could have something serious to deploy at all :grin:

**Reply by objeck** [reply]
Can you zip and post the code?
